greatestPalindrome.js

```diff
@@ -0,0 +1,56 @@
+
+/*Given a string return the longest palindrome that can be constructed by removing or shuffling characters. 
+Example: 
+'aha' -> 'aha' 
+'ttaatta' -> ' ttaaatt' 
+'abc' -> 'a' or 'b' or 'c' 
+'gggaaa' -> 'gaaag' or 'aggga' 
+Note if there are multiple correct answers you only need to return 1 palindrome.
+*/
+var word = "aabczRc";
+var wordDict = {};
+var leftPalindrome = [];
+var rigthPalindrome = [];
+var palindrome;
+
+function dictionaryCreator(word){
+  for(var index in word){
+    var letter = word[index];
+    if(wordDict[letter]){
+      wordDict[letter]++;
+    }
+    else{
+      wordDict[letter] = 1;
+    }
+  }
+};
+
+dictionaryCreator(word)
```

Comment:
the  code that runs the functions or logs the result should be at the end of the file always